{{ cookiecutter.package_name }}/Makefile

```diff
@@ -1,25 +1,27 @@
 .DEFAULT_GOAL := help
 .PHONY: docs
 SRC_DIRS = ./{{ cookiecutter.module_name }}
-BLACK_OPTS = --exclude templates ${SRC_DIRS}
 
 # Warning: These checks are not necessarily run on every PR.
 test: test-lint test-types test-format  # Run some static checks.
 
 test-format: ## Run code formatting tests
```

Comment:
let's add build and test pythonpackage make recipes as done in other PRs.

Reply:
We discussed this before with @regisb and @kdmccormick and decided to keep the cookiecutter as simple as possible.